lib/tasks/admintasks.rake

```diff
@@ -0,0 +1,24 @@
+namespace :admin do
+  desc 'Make a new admin'
+  task new: :environment do
```

Comment:
@akshay1y2 read about, why we need :environment here in the task. What if we don't use this :envionment